### ISSUE TITLE
WebSocket connections are not allowed or blocked

### ISSUE THREAD

**Issue by Baker68** [ulixee/hero#141]
Code : 
```
const Hero = require('@ulixee/hero-playground');

(async () => {
    const hero = new Hero({showChrome: true, showChromeInteractions: true});

    await hero.goto('https://libwebsockets.org/testserver/');
    await hero.waitForPaintingStable();
    // await hero.close();
})();
```
![image](https://user-images.githubusercontent.com/34166422/184476711-95b0ab67-9dd3-401b-aefa-80dfc4993374.png)
If I reload the page from the browser : 
![image](https://user-images.githubusercontent.com/34166422/184476792-dbc9abb1-3e6e-44ef-a18a-7baf8aaf2637.png)

**Comment by blakebyrnes** [comment]
Thanks for reporting. Looking into this.

**Comment by blakebyrnes** [comment]
Fixed in next version of Agent (https://github.com/unblocked-web/agent/pull/11/commits/cca0419f65539129d5f5a44e76f2ce4ecdec6b24)

**Comment by blakebyrnes** [comment]
Fixed in version 2.0.0-alpha.10
